src/amicableNumbers/Duplicates.java

```diff
@@ -0,0 +1,36 @@
+package amicableNumbers;
+
+import java.util.ArrayList;
+
+public class Duplicates {
+	private static ArrayList<String> letterList;
+	public static void main(String[] args) {
+		letterList = new ArrayList<String>();
+		letterList.add("a");
+		letterList.add("a");
+		letterList.add("a");
+		letterList.add("b");
+		letterList.add("c");
+		letterList.add("c");
+		letterList.add("d");
+		letterList.add("d");
+		
+		System.out.println(letterList);
+		removeDuplicates(letterList);
+		System.out.println(letterList);
+		
+	}
+	
+	public static void removeDuplicates(ArrayList<String> list) {
+		if(list.size() > 1) {
+			for(int x = 0; x < list.size()-1; x++) {
+				String y = list.get(x);
+				while(list.subList(x+1, list.size()).contains(y)) {
+					list.remove(y);
+				}
+			}
+		}
```

Comment:
This works perfectly fine but would be slow. If you know what a big O is, the big O of this function you wrote is O(n^2). There would be a more efficient way to run this that uses an ArrayList to store all the checked ones. The O is O(n) for that function.
Also, watch out for the shift of index after you remove something.